src/main/main.ino

```diff
@@ -5,10 +5,18 @@
 */
 
 #include <Arduino.h>
+#include <Servo.h>
```

Comment:
this will cause the compilation to fail, if we have referenced the ESP servo in the steering file, then redundant declaration will cause confusion and issues, remove it please.